src/Bicep.LangServer/Snippets/Templates/resource-child-defaults.bicep

```diff
@@ -1,6 +1,6 @@
 ﻿// Child Resource with defaults
-resource ${1:Identifier} 'Microsoft.${2:Provider}/${3:ParentType}/${4:ChildType}@${5:Version}' = {
-  name: $6
+resource ${1:Identifier} 'Microsoft.${2:Provider/ParentType/ChildType@Version}' = {

```

Comment:
Since we're now treating the whole type string as one placeholder, it doesn't feel like there's any benefit to keeping the `Microsoft.`. Should we change this to just:
```
resource ${1:Identifier} '${2:Provider/ParentType/ChildType@Version}' = {
```

Reply:
Makes sense. Updated it here:
https://github.com/Azure/bicep/commit/584a25f1acf10c2673dd2795f9cfcd833c885130